tests/sklearn/test_SklearnNaiveBayesConverter.py

```diff
@@ -18,6 +18,12 @@ def _fit_model_binary_classification(self, model):
         model.fit(X, y)
         return model, X.astype(numpy.float32)
 
```

Comment:
This file no longer exists in the master branch due to the refactoring. You might want to pull from master, or consider moving these changes to the skl2onnx repo!